h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java

```diff
@@ -800,7 +808,20 @@ private void parseProperties(JsonParser jp, Object[] values) throws IOException,
                 values[cachedColumnIndex.get(fieldName)] =  jp.getValueAsDouble();
             } else if (value == JsonToken.VALUE_NUMBER_INT) {
                 values[cachedColumnIndex.get(fieldName)] =  jp.getBigIntegerValue();
-            } else {
+            } else if (value == JsonToken.START_ARRAY) {
+                ArrayList<Object> arrayList = parseArray(jp);
+                Object[] array = new Object[arrayList.size()];
```

Comment:
You can do simpler by using `List.toArray()` method which return the Object array corresponding to the list.